src/usr/local/buildpack/utils/cache.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# will attempt to download the file at the given url and stores it in the cache.
+# If this file already exists, will return the cached version
+# The cache will only used if BUILDPACK_CACHE_DIR is set
+# First argument is the url, second one is the filename (optional)
+function get_from_url () {
+  local url=${1}
+  check url true
+
+  local name
+  name=${2:-$(basename "${url}")}
```

Comment:
Thisd works for urls? 😏

Reply:
yes, urls are not that different than a file path and for our use case this is more than enough